road/validation_road.py

```diff
@@ -0,0 +1,113 @@
+import os
+import pandas as pd
+from simpledbf import Dbf5
+import configparser
+import json
+import yaml
+from sklearn.linear_model import LinearRegression
+from io import StringIO
+import geopandas as gpd
+from scatter import *
+
+
+config = configparser.ConfigParser()

```

Comment:
move config readings down to main
(unless if any of these are global variables---in which case, do not have global variables, instead pass them directly into the individual functions